dump/main.c

```diff
@@ -140,9 +140,68 @@ static int dump_full(const char* spec, bool used_sectors)
 	return 0;
 }
 
+static int dump_file(const char* spec, const char* path)
+{
+	struct exfat ef;
+	struct exfat_node* node;
+	cluster_t cluster, next_cluster;
+	cluster_t start_cluster = 0;
```

Comment:
Better use `EXFAT_CLUSTER_END` constant instead of magic 0 (which is in fact `EXFAT_CLUSTER_FREE`).